src/qml/WelcomeScreen.qml

```diff
@@ -86,8 +86,10 @@ Page {
         Button {
           id: localProjectButton
           padding: 8 * dp
+/* Not available on Qt 5.11 (OSGeo4W)
```

Comment:
@nirvn do you see a problem here?

Reply:
Could you try to adjust the padding or margin values to achieve same size?

Reply:
I suspect an update soon tbh https://lists.osgeo.org/pipermail/qgis-developer/2020-January/060067.html